tests/lookup/models.py

```diff
@@ -19,6 +20,7 @@ def __str__(self):
 class Author(models.Model):
     name = models.CharField(max_length=100)
     alias = models.CharField(max_length=50, null=True, blank=True)
+    date = models.DateField(default=datetime.date.today())
```

Comment:
Remove `()` -- that's a fixed default which gives a check warning when running the tests.